dashboard/public/pixelation/pixelation.js

```diff
@@ -438,6 +445,18 @@ function setSliders() {
   drawGraph();
 }
 
+function setMinTextValues() {
+  if (heightText.value < 1) {
+    heightText.value = 1;
+  }
+  if (widthText.value < 1) {
+    widthText.value = 1;
+  }
+  if (bitsPerPixelText.value < 1) {
+    bitsPerPixelText.value = 1;
+  }
+}
```

Comment:
I can't break it in Chrome, but it may break in another browser. I've changed it to use the ?: operator.